.changeset/khaki-cloths-repair.md

```diff
@@ -0,0 +1,30 @@
+---
+'astro': minor
+---
+
+Introduces a `priority` attribute for the `Image` component.
```

Comment:
```suggestion
Adds a new `priority` attribute for Astro's image components.
```
We'll always go with simpler English when we can! (And yes, "adds" and "new" are redundant together, but for reaching the brain of skimmers, it's sometimes helpful, so it's an option!)

Also, if we're not saying `<Image />` and `<Picture />`, then do you think it's ok to lower case refer generically to them both as "Astro's image components"? We don't have a good generic phrase that we've settled on yet.